src/ImageService.Data/ImageMessageRepository.cs

```diff
@@ -46,14 +45,12 @@ public bool Remove(List<Guid> imageIds)
         return false;
       }
 
-      SqlCommand command = new();
-
       foreach (Guid imageId in imageIds)
       {
-        command.CommandText = $@"DELETE FROM {DbImageMessage.TableName} WHERE Id = '{imageId}' OR ParentId = '{imageId}' OR
+        string query = $@"DELETE FROM {DbImageMessage.TableName} WHERE Id = '{imageId}' OR ParentId = '{imageId}' OR
             Id IN (SELECT ParentId FROM ImagesProjects WHERE Id = '{imageId}' AND ParentId IS NOT NULL);";
 
-        _provider.ExecuteRawSql(command.CommandText);
+        _provider.ExecuteRawSql(query);
```

Comment:
you can paste queryString in arguments without variable

Reply:
done